### DISCUSSION TITLE
Centrar texto flex box

### DISCUSSION THREAD

**Discussion by AndrjsM** [Launch-X-Latam/MisionFrontEnd#321] [Q&A]
Hola, chicos un gusto saludarles.

La cosa es que me está dando por la madre centrar el texto en un elemento hijo y quería ver si alguien me puede orientar sobre la mejor solución.

La maquetación es la siguiente:
Nav barra, que contiene 2 elementos (hijo)
1. al título de mi página (podría ser el logo).
2. al menú (que a su vez va a contener 4 opciones)

El menú a su vez tiene un section dentro que lo hice un contenedor padre, dentro del hijo del contenedor nav, por que? bueno para poder acomodar los botones del menú, hasta ahí todo bien, a los hijos los hijos los alineo con stretch porque quiero que se ajusten al 100% de su contendor sin dimensionar (px) esto para que sea responsive pero ahí viene el problema, el texto no se centra verticalmente, lo intente de todo y no lo logre.

Quiero que la etiqueta a del texto abarque el 100% de su contendor porque quiero darle formato de botón y un poco de diseño.

Les adjunto la página y el código: (nota, le puse colores x para que se note lo que les mencione)
https://andrjsm.github.io/Practica_pasteler-a/
https://github.com/AndrjsM/Practica_pasteler-a

Estaba pensando meter un div dentro de los contenedores para darle formato de tabla y poder usar vertical-align: middle; pero no se si sea recomendado por limpieza del código o si funcione jaja

**Comment by AlessandrohL** [comment]
Intentaste con flexbox?

Contenedor padre:

_display: flex;_
_flex-direction: column;_
_Justify-content: center;_
_align-items: center;_

Con eso deberia funcionar

**Comment by doguedogue** [comment]
Para alinear filas
```align-items: center;```

Para alinear con bootstrap agregas la clase
```text-center```

Para alinear en maquina de escribir
``` Cuentas la cantidad de letras, el resultado lo restas a 85 (tamaño hoja) y lo divides entre dos, ahi empiezas a escribir```
